samples/photo_album/src/reducers/PhotosListReducer.js

```diff
@@ -0,0 +1,16 @@
+const PhotosListReducer = (photos = [], action) => {
+    switch (action.type) {
+        case 'PHOTOS_FETCHED':
+            return action.photos;
```

Comment:
Should return an immutable object

Reply:
This comment was probably overlooked. The returned object should be immutable (at least return a copy of action.photos)